lib/addons/p5.dom.js

```diff
@@ -1043,281 +1148,4 @@
       p5.prototype.set.call(this, x, y, imgOrCol);
     }
   };
```

Comment:
That's really weird... It looks like I'm working with v.0.2.0 ... my `upstream` remote is set to processing/p5.js but when I do `git pull upstream master ` it doesn't seem to update properly?